docs/3.0.0-beta.x/concepts/configurations.md

```diff
@@ -442,19 +442,14 @@ The session doesn't work with `mongo` as a client. The package that we should us
 - `host` (string): Host name. Default value: `localhost`.
 - `port` (integer): Port on which the server should be running. Default value: `1337`.
 - `emitErrors` (boolean): Enable errors to be emitted to `koa` when they happen in order to attach custom logic or use error reporting services.
-- `proxy`
-  - `enabled` (boolean): Enable proxy support such as Apache or Nginx. Default value: `false`.
-  - `ssl` (boolean): Enable proxy SSL support.
-  - `host` (string): Host name your proxy service uses for Strapi.
-  - `port` (integer): Port that your proxy service accepts connections on.
-- [`cron`](https://en.wikipedia.org/wiki/Cron)
-  - `enabled` (boolean): Enable or disable CRON tasks to schedule jobs at specific dates. Default value: `false`.
+- `server`
+  - `url` (string): Url of the server. Enable proxy support such as Apache or Nginx. Default value: `http://${host}:${port}`.
 - `admin`
-  - `autoOpen` (boolean): Enable or disabled administration opening on start. Default value: `true`.
-  - `path` (string): Allow to change the URL to access the admin panel. Default value: `/admin`.
+  - `url` (string): Url of your admin panel. Default value: `/admin`. Note: If the url is relative, it will be concatenate with `server.url`.
```

Comment:
Combined with previous note

- `/admin`
- `http://${host}:${port}/admin` (to not have `http://${host}:${port}/api/admin`)

Reply:
I'm not sure of what you mean, especialy with `to not have http://${host}:${port}/api/admin`. Can you be more specific?